src/component/MainPage.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+class MainPage extends React.Component {
```

Comment:
What the hack happened to MainPage component? Why it not return anything? Is it ok?

Reply:
Fixed render for MainPage